### DISCUSSION TITLE
env icons not showing

### DISCUSSION THREAD

**Discussion by Muzammil-hasan** [kyazdani42/nvim-web-devicons#473] [Q&A]
![Screenshot 2024-06-10 231033](https://github.com/nvim-tree/nvim-web-devicons/assets/70757697/357e1c86-000f-49ff-ad13-34929c86727a)
Icons are not showing for .env.* files

**Comment by alex-courtis** [comment]
Those file names are not mapped, only `".env"` itself.

If those files have a vim filetype set you could add a mapping via [set_icon_by_filetype](https://github.com/nvim-tree/nvim-web-devicons?tab=readme-ov-file#getting-and-setting-icons-by-filetype)

Something like `require("nvim-web-devicons").set_icon_by_filetype { env = "env", }`
